python/paddle/fluid/optimizer.py

```diff
@@ -345,11 +352,16 @@ class SGDOptimizer(Optimizer):
             sgd_optimizer.minimize(cost)
     """
 
-    def __init__(self, learning_rate, regularization=None, name=None):
+    def __init__(self,
+                 learning_rate,
+                 regularization=None,
+                 weight_decay=None,
+                 name=None):
```

Comment:
1. Add comments for these classes.
2. weigth_decay can receive a float value as coeff.